recipes/credstash/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "credstash" %}
+{% set version = "1.15.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 814560f99ae2409e2c6d906d878f9dadada5d1d0a950aafb6b2c0d535291bdfb
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - credstash = credstash:main
+
+requirements:
+  host:
+    - python
+    - pip
+    - nose >=1.3.6
```

Comment:
This is unusual, put is added as `setup_requires` in upstream's [setup.py](https://github.com/fugue/credstash/blob/56df8e051fc4c8d15d5e7e373e88bf5bc13f3346/setup.py#L32-L34) file.